scripts/common/preflights.sh

```diff
@@ -411,6 +411,18 @@ function host_preflights() {
         opts="${opts} --is-upgrade"
     fi
 
+    # Remove previous file if it exists
+    if [ -f "$VENDOR_PREFLIGHT_SPEC" ]; then
+      rm "$VENDOR_PREFLIGHT_SEC"
```

Comment:
SEC should be SPEC, the `rm` statement doesn't do anything because the variable name is incorrect

Reply:
good catch